dashboard/app/views/admin_nps/nps_form.html.haml

```diff
@@ -0,0 +1,15 @@
+%h1 NPS Survey
+
+%p User the dropdown menu below to determine the audience of the most recent NPS survey.
+%p This will save as a DCDO flag, which will automatically update the audience.
+%p NOTE: Any audience other than "none" will launch the survey immediately.
+
+%h2 Survey Audience
+
+= form_tag({controller: 'admin_nps', action: 'nps_update', method: "post"}, style: 'margin: 5px;') do
+  %select{:name => "audience"}
```

Comment:
(minor) would it be a pain to get the dropdown and the save button to line up a bit better?

Reply:
I was able to use a table-caption styling to get the save to sit below the dropdown and left-align it. The photos in the pr are updated accordingly